### PR TITLE
[connectivity_plus] Update connectivity_plus to 6.1.0

### PR DESCRIPTION
**Pull request by JSUYA**
- Update connectivity_plus to 6.1.0.
- Update connectivity_plus_platform_interface to 2.0.1.
- 
https://github.com/flutter-tizen/plugins/issues/758

### CODE REVIEW

File: packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc [hunk 1]
```diff
@@ -117,7 +120,10 @@ class ConnectivityPlusTizenPlugin : public flutter::Plugin {
       Connection connection;
       ConnectionType type = connection.GetType();
       if (type != ConnectionType::kError) {
-        result->Success(flutter::EncodableValue(ConnectionTypeToString(type)));
+        flutter::EncodableList encodedList;
```
Comment (xiaowei-guan) [inline packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc:123]: Suggest changing  encodedList to encoded_list

Comment (JSUYA) [reply packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc]: Thank you. I updated PR

File: packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc [hunk 2]
```diff
@@ -52,7 +52,10 @@ class ConnectivityStreamHandler : public FlStreamHandler {
 
     ConnectionTypeCallback callback = [this](ConnectionType type) -> void {
       if (type != ConnectionType::kError) {
-        events_->Success(flutter::EncodableValue(ConnectionTypeToString(type)));
+        flutter::EncodableList encodedList;
```
Comment (xiaowei-guan) [inline packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc:55]: Suggest changing encodedList to encoded_list. Sorry,I didn't see that.

Comment (JSUYA) [reply packages/connectivity_plus/tizen/src/connectivity_plus_tizen_plugin.cc]: Oh my fault, Thanks!

**Review by xiaowei-guan — APPROVED** [review]

**Pull request merged**